src/assets/pDmgDealer.java

```diff
@@ -1,7 +1,10 @@
 package assets;
 /*
- * used to group plants that deal dmg
+ * used to group plants that deal have power to kill zombies
  */
 public class pDmgDealer extends Plant {
```

Comment:
I don't think we need this unless Dmg Dealers have different attributes from plants. maybe dmg should be moved from plant to this class?